app/src/main/java/swati4star/createpdf/fragment/MergeFilesFragment.java

```diff
@@ -233,6 +234,11 @@ void mergeFiles(final View view) {
                         StringUtils.getInstance().showSnackbar(mActivity, R.string.snackbar_name_not_blank);
                     } else {
                         if (!mFileUtils.isFileExist(input + getString(R.string.pdf_ext))) {
+                            File docsFolder = new File(mHomePath);
```

Comment:
Can you also explain what the issue was? In which cases were you not able to use the "Select Files" button.

The intent here is, I want to check if this fix is required for other features (like, split PDF etc also). If yes, we need to make this change in a central place,